internal/report/report_test.go

```diff
@@ -42,39 +43,63 @@ func TestSource(t *testing.T) {
 	for _, tc := range []testcase{
 		{
 			rpt: New(
-				testProfile.Copy(),
+				testProfile1.Copy(),
 				&Options{
 					OutputFormat: List,
 					Symbol:       regexp.MustCompile(`.`),
 					TrimPath:     "/some/path",
 
 					SampleValue: sampleValue1,
-					SampleUnit:  testProfile.SampleType[1].Unit,
+					SampleUnit:  testProfile1.SampleType[1].Unit,
 				},
 			),
 			want: path + "source.rpt",
 		},
 		{
 			rpt: New(
-				testProfile.Copy(),
+				testProfile1.Copy(),
 				&Options{
 					OutputFormat: Dot,
 					CallTree:     true,
 					Symbol:       regexp.MustCompile(`.`),
 					TrimPath:     "/some/path",
 
 					SampleValue: sampleValue1,
-					SampleUnit:  testProfile.SampleType[1].Unit,
+					SampleUnit:  testProfile1.SampleType[1].Unit,
 				},
 			),
 			want: path + "source.dot",
 		},
+		{
+			rpt: New(
+				testProfile2.Copy(),
+				&Options{
+					OutputFormat: Dot,
+					CallTree:     true,
+					Symbol:       regexp.MustCompile(`.`),
+					TrimPath:     "/some/path",
```

Comment:
What do you specify TrimPath in this test for?

Reply:
Removed this.